corehq/apps/userreports/templates/userreports/configurable_report.html

```diff
@@ -6,133 +6,7 @@
     <script src="{% static 'reports/js/reports.util.js' %}"></script>
     <script src="{% static 'reports/js/reports.config.js' %}"></script>
     <script src="{% static 'reports/js/saved_reports.js' %}"></script>
-{% endblock %}
-
-{% block js-inline %}{{ block.super }}
-    <script>
-        $(function(){
-            var $editReportButton = $("#edit-report-link");
-
-            {% if report.spec.report_meta.created_by_builder %}
-
-            var $applyFiltersButton = $("#apply-filters");
-            $applyFiltersButton.click(function(){
-                window.analytics.usage("Report Viewer", "Apply Filters", '{{ report.spec.report_meta.builder_report_type }}');
-            });
-            gaTrackLink($editReportButton, 'Report Builder', 'Edit Report', '{{ report.spec.report_meta.builder_report_type }}');
-            window.analytics.trackWorkflowLink($editReportButton, "Clicked Edit to enter the Report Builder");
-            window.analytics.usage("Report Viewer", "View Report", '{{ report.spec.report_meta.builder_report_type }}');
-
-            {% else %}
-            gaTrackLink($editReportButton, 'Edit UCR', 'Edit UCR');
-            {% endif %}
-
-            {% for event in report_builder_events %}
-            window.analytics.usage.apply(this, {{ event|JSON }});
-            {% endfor %}
-        });
-    </script>
-    {% if not report.is_static %}
-    <script>
-        $(function(){
-            // Poll the status of the data source
-            var retrying = false;
-            (function poll() {
-                $.ajax({
-                    url: "{% url 'configurable_data_source_status' domain report.spec.config_id %}",
-                    dataType: 'json',
-                    success: function(data) {
-                        if (data.isBuilt){
-                            $('#built-warning').addClass('hide');
-                            if (retrying){
-                                location.reload();
-                            } else if ($('#report-filters').find('.control-label').length === 0) {
-                                $('#report-filters').submit();
-                            }
-                        } else {
-                            retrying = true;
-                            $('#built-warning').removeClass('hide');
-                            setTimeout(poll, 5000);
-                        }
-                    }
-                });
-            })();
-        });
-    </script>
-    {% endif %}
-    <script>
-        var urlSerialize = hqImport('reports/js/reports.util.js').urlSerialize;
-        var standardHQReport = new HQReport({
```

Comment:
@orangejenny this breaks the Save button in the New Saved Report dialog that pops up when you click Save... on the page of a configurable report.

The reason it breaks is that saved_report.js relies on the existence of a `standardHQReport` global variable. I don't have an obvious suggestion for how to fix, it was basically using global variables to emulate a kind of plugin system, where that dialog would work with whatever report happened to be on the page rather than having to be passed the report during initialization. Maybe you're more familiar there's a convention already used that we can tap into, but it seems the easiest fix is for such a variable to be available from the global context (even if it's not actually a global variable) by having the report js do something like `setPageReport(standardHQReport)` and `saved_reports.js` doing something like `getPageReport()`.

This is likely broken in the analogous page on all reports.